go/grpcwebproxy/main.go

```diff
@@ -152,6 +152,7 @@ func buildGrpcProxyServer(logger *logrus.Entry) *grpc.Server {
 		outCtx, _ := context.WithCancel(ctx)
 		mdCopy := md.Copy()
 		delete(mdCopy, "user-agent")
+		delete(mdCopy, "connection")
```

Comment:
Sorry, could you just add a comment with a reference to the issue as well for future readers?

Reply:
Comment added!

Reply:
Ah, I mean literally a link to the issue, like this: https://github.com/improbable-eng/grpc-web/issues/568. In addition to the explanation here.

Reply:
Makes sense, it's been added :)